UPGRADE-4.0.md

```diff
@@ -25,6 +25,8 @@ If you have implemented a custom admin, you must adapt the signature of the foll
  * `getDashboardActions`
  * `getActionButtons`
  * `isCurrentRoute`
+ 
+The method signature of `configureActionButtons` has changed. A new parameter `actionList` was added.
```

Comment:
IMO it's not necessary because you can look at the interface to see the difference.